### PR TITLE
Update UserProfile.tsx

### PR DESCRIPTION
**Pull request by jjlange**

### CODE REVIEW

**Comment by jjlange** [comment]
pls accept

**Comment by jjlange** [comment]
with sugar on top

**Comment by nullaf** [comment]
Lgtm

File: src/screens/UserProfile/UserProfile.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 
 export function UserProfile() {
-  return <div />;
+  return <div></div>;
```
Comment (karinacrmcl) [inline src/screens/UserProfile/UserProfile.tsx:4]: image

Empty components are self-closing  react/self-closing-comp[](url)

**Comment by jjlange** [comment]
thank yu!! 🥹

**Comment by CyberKatze** [comment]
looks too complex, it's out of my competence

**Pull request closed without merging**